dynawo/sources/Models/Modelica/Dynawo/Examples/Nordic/Grid/BaseClasses/NetworkWithAlphaBetaLoads.mo

```diff
@@ -15,101 +15,100 @@ within Dynawo.Examples.Nordic.Grid.BaseClasses;
 
 model NetworkWithAlphaBetaLoads "Nordic test grid with buses, lines, shunts and voltage-dependent loads"
   import Modelica.ComplexMath;
-  import Modelica.SIunits;
   import Dynawo;
   import Dynawo.Electrical;
   import Dynawo.Types;
 
   extends Dynawo.Examples.Nordic.Grid.BaseClasses.Network;
 
-  Electrical.Loads.LoadAlphaBeta load_01(alpha = 1, beta = 2, s0Pu = s0Pu_Load_01, u0Pu = u0Pu_Load_01, i0Pu = i0Pu_Load_01) annotation(
```

Comment:
You can have a generic parameter alpha and a generic parameter beta to be used for all the loads. alpha = 1 and beta = 2 will be given in the test case

Reply:
Done